content/blog/gatsby-blog/index.md

```diff
@@ -0,0 +1,323 @@
+
+There are tons of alternatives but if you already host your page's code in github
+then this will be pretty easy to setup.
+
+Follow the instructions on [the website](https://pages.github.com/) to enable github pages.
+
+These are the extra steps I took for this blog
+
+### 1 make `develop` your default branch (only for _project site_)
+
+Since this blog qualifies as a *project site* (per repo) instead of a *user or organization site*
+we need to have the following repo branch configuration:
+
+- `develop`: where our gatsby code lives
+- `master`: where our gatsby-generated static files for our website are stored.
+
+These generated static files will be the ones that github pages will deploy and show.
+
+Have in mind that you will be managing two code bases, one is your gatsby site, the code that you will
+be interacting with as a developer and then we have the generated static site code that will be the one
+that we serve.
+Think of these two code bases as the source code and the compiled target code in traditional
+compiled programming languages such as C/C++ or Rust.
+
+### 2 Adjust the `package.json` deploy script
+
+It should look like this
+
+```json
+"deploy": "gatsby build --prefix-paths && gh-pages -d public -b master"
+```
+
+- `--prefix-paths` is only useful if you are going to use the default github domain: `username.github.io/reponame`. Remove it if you are making your site accessible from some `domain.com`
+- `gh-pages` is a helper node package that you need to install: `npm install --dev gh-pages`
+- `-b master` specifies to `gh-pages` package that you want to deploy to the `master` branch
+
+
+### 3 Adjust your `pathPrefix`
+
+IMPORTANT! only do this if you are using the default github domain. If you are using a custom domain,
+like `myblog.com` you should probably leave it unset, or if you want your blog to live in a custom path
+of your custom domain like `myblog.com/custompath` then adjust accordingly.
+
+
+Go to `gatsby-config.js` and set the top level property 
+
+```javascript
+pathPrefix: reponame
+```
+
+
+
+### 4 deploy!
+
+```bash
+
+# make sure you commit and push your changes to develop
+git commit ...
+git push origin develop
+
+# deploy!
+npm run deploy
+```
+
+Now if you go to `username.github.io/reponame` you should be able to navigate your site!
+
+Congratulations! your site is live!
+
+
+## [optional] Buy a domain and use it
+
+We used `godaddy` but we have no affiliations with them and there are plenty of
+domain name providers. Choose the right one for you, buy your domain and then adjust DNS accordingly.
+
+
+### 1 Setup your DNS
+
+Most of the domain providers will let you set up the DNS records for your site. We are going to use
+that to redirect to your github page, set up the following records in your DNS managing tool.
+
+Read the official [instructions](https://help.github.com/en/github/working-with-github-pages/about-custom-domains-and-github-pages).
+
+But here is the gist:
```

Comment:
I don't think I've ever seen the word "gist" by itself, it is usually accompanied by "of [x]". I think it sounds better as well.
https://www.merriam-webster.com/dictionary/gist
https://www.grammarly.com/blog/gist-or-jist/

Reply:
it is usually used like this, it is probably something in dev related posts, but I have also seen it used in conversations, can I leave it?